packages/next/export/index.ts

```diff
@@ -215,6 +215,9 @@ export default async function(
     hotReloader: null,
     canonicalBase: (nextConfig.amp && nextConfig.amp.canonicalBase) || '',
     isModern: nextConfig.experimental.modern,
+    ampValidator:
+      (nextConfig.experimental.amp && nextConfig.experimental.amp.validator) ||
+      '',
```

Comment:
I don't think defaulting to `''` is correct, it should probably be `undefined`, considering https://github.com/ampproject/amphtml/blob/master/validator/nodejs/index.js#L296